octree/include/pcl/octree/octree_pointcloud_adjacency.h

```diff
@@ -247,9 +247,8 @@ namespace pcl
 
 }
 
-//#ifdef PCL_NO_PRECOMPILE
+// Note: Don't precompile this octree type to speed up compilation. It's probably rarely used.
```

Comment:
This octree type drives the [Supervoxel Clustering](http://pointclouds.org/documentation/tutorials/supervoxel_clustering.php) algorithm, which is extremely popular according to my feelings.